build/pr-netfx-validation.yaml

```diff
@@ -57,13 +57,13 @@ stages:
         displayName: 'If above is partially succeeded, then fail'
         condition: eq(variables['Agent.JobStatus'], 'SucceededWithIssues')
 
-  # - template: templates/azure-pipeline.template.yaml
-  #   parameters:
-  #     name: 'netfx_tests_windows'
-  #     displayName: 'Unit Tests (Windows 2019)'
-  #     vmImage: 'windows-2019'
-  #     scriptFileName: build.cmd
-  #     scriptArgs: RunTestsNetFx471 #Add ' incremental' to Run tests incremental-ly
-  #     outputDirectory: 'TestResults'
-  #     artifactName: 'netfx_tests_windows-$(Build.BuildId)'
-  #     packageFeed: $(packageFeed)
\ No newline at end of file
+  - template: templates/azure-pipeline.template.yaml
+    parameters:
+      name: 'netfx_tests_windows'
+      displayName: 'Unit Tests (Windows 2022)'
+      vmImage: 'windows-2022'
+      scriptFileName: build.cmd
+      scriptArgs: RunTestsNetFx471 #Add ' incremental' to Run tests incremental-ly
```

Comment:
This is needed. Max added those in his migration PR

Reply:
That's OK - The question is why? We already have 2 of them and wonder why we need this; is it because of https://msazure.visualstudio.com/One/_workitems/edit/26682694/?view=edit?